sql.go

```diff
@@ -83,27 +83,30 @@ func (u *NullUUID) Scan(src interface{}) error {
 	return u.UUID.Scan(src)
 }
 
+var nullJSON = []byte("null")
```

Comment:
Why declare a package-level variable that's only referenced once?

Reply:
`nullJSON` is declared here so that `NullUUID.MarshalJSON()` does not allocated when it nil/invalid. This is common practice: [`encoding/json/decode.go#L602`](https://github.com/golang/go/blob/9f0234214473dfb785a5ad84a8fc62a6a395cbc3/src/encoding/json/decode.go#L602).